packages/terra-section-header/tests/jest/SectionHeader.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import SectionHeader from '../../src/SectionHeader';
+
+describe('SectionHeader', () => {
+  const defaultRender = <SectionHeader title="foo" />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const wrapper = shallow(defaultRender);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render without an accordion icon when no \'onClick()\' is passed', () => {
```

Comment:
In cases like this. I would just use double quotes for the string, and then just use the single quote when needed.

Reply:
Done 13b006165da82994ac8978f8de4dc8a74f74b4d0